pkg/kubectl/cmd/set/set_serviceaccount.go

```diff
@@ -127,16 +127,16 @@ func (saConfig *serviceAccountConfig) Complete(f cmdutil.Factory, cmd *cobra.Com
 	saConfig.serviceAccountName = args[len(args)-1]
 	resources := args[:len(args)-1]
 	includeUninitialized := cmdutil.ShouldIncludeUninitialized(cmd, false)
-	builder := f.NewBuilder().ContinueOnError().
+	builder := f.NewBuilder().
+		LocalParam(saConfig.local).
+		ContinueOnError().
```

Comment:
intentional addition?  I like it, but wanted to make sure it was on purpose.

Reply:
Yes, was down below.

Reply:
I have this TODO somewhere, but local shouldn't be a command specific flag.  It should be a global and enforced at multiple levels.  There are decisions that Factory methods make that should depend on local.